pki.spec

```diff
@@ -422,9 +422,9 @@ Requires:         glassfish-jaxb-api
 Requires:         slf4j
 Requires:         slf4j-jdk14
 Requires:         jpackage-utils >= 0:1.7.5-10
-Requires:         jss = 4.9
-Requires:         ldapjdk = 4.23
-Requires:         %{product_id}-base = %{version}-%{release}
+Requires:         jss >= 4.9.6, jss < 5.0
+Requires:         ldapjdk >= 4.23
+Requires:         %{product_id}-base >= %{version}-%{release}
```

Comment:
For this one we probably should use `=` since it comes from the same package, so we want to use the exact same version and release number:
```
Requires:         %{product_id}-base >= %{version}-%{release}
```